.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+enable-beta-ecosystems: true
+updates:
+    - package-ecosystem: npm
```

Comment:
In project used the ```pnpm```

Reply:
![Uploading Capture d’écran 2023-06-23 à 10.12.47.png…]()

Reply:
<img width="733" alt="Capture d’écran 2023-06-23 à 10 12 47" src="https://github.com/Pisyukaev/twitch-sub-goal/assets/97875033/c07c862a-de41-4618-8e95-ff6717f4c460">